libpod/runtime.go

```diff
@@ -210,6 +210,10 @@ func newRuntimeFromConfig(ctx context.Context, conf *config.Config, options ...R
 	}
 
 	if err := shutdown.Register("libpod", func(sig os.Signal) error {
+		// For `systemctl stop podman.service` support, exit code should be 0
+		if sig == syscall.SIGTERM {
```

Comment:
This doesn't seem right - I'm trying a lot of longrunning commands and hitting them all with SIGTERM, and none of them exit cleanly?

Reply:
`systemctl stop podman.service` always returned 1, after this change it now returns 0 as expected.